spec/lib/coach/request_serializer_spec.rb

```diff
@@ -34,5 +36,38 @@
         expect(sanitized).to eq('value')
       end
     end
+
+    context "as an instance" do
+      let(:mock_request) do
+        instance_double("Request", :request, format:              nil,
```

Comment:
I would favour creating an instance double of `ActionDispatch::Request` here so that it verifies we're not stubbing anything that isn't actually defined on that class.